packages/fx-core/src/component/generator/constant.ts

```diff
@@ -10,7 +10,7 @@ export const defaultTimeoutInMs = 20000;
 export const sampleDefaultTimeoutInMs = 40000;
 export const templateFileExt = ".tpl";
 export const templateAlphaVersion = "templates-0.0.0-alpha";
-export const templateBetaVersion = "templates-0.0.0-beta";
+export const templateRcVersion = "templates@0.0.0-rc";
```

Comment:
I think we don't need to hard code this version since we decide to reuse config.version for both v2 templates and v3 templates.